dtos/reading.go

```diff
@@ -287,6 +287,7 @@ func (b BaseReading) Validate() error {
 func ToReadingModel(r BaseReading) models.Reading {
 	var readingModel models.Reading
 	br := models.BaseReading{
+		Id:           r.Id,
```

Comment:
Is this an unrelated bug that we need to submit issue and the fix in different PR?

Reply:
Yes, to handle this bug, I submitted a separate issue at https://github.com/edgexfoundry/go-mod-core-contracts/issues/673